cmd/kaspawallet/broadcast.go

```diff
@@ -28,27 +28,28 @@ func broadcast(conf *broadcastConfig) error {
 		return errors.Errorf("Both --transaction and --transaction-file cannot be passed at the same time")
 	}
 
-	transactionHex := conf.Transaction
+	transactionsHex := conf.Transaction
```

Comment:
Did that (also for TransactionsFile), but left the cli flag to still say "--transaction" in singular, because the user should be transparent to the split at this point.